### PR TITLE
COMMERCE-10466 added channel infos to mini cart

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-10466

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/src/main/java/com/liferay/headless/commerce/delivery/catalog/internal/dto/v1_0/converter/ChannelDTOConverter.java [hunk 1]
```diff
@@ -43,7 +43,7 @@ public Channel toDTO(DTOConverterContext dtoConverterContext)
 		throws Exception {
 
 		CommerceChannel commerceChannel =
-			_commerceChannelService.getCommerceChannel(
+			_commerceChannelLocalService.getCommerceChannel(
```
Comment (alessiorendina) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/src/main/java/com/liferay/headless/commerce/delivery/catalog/internal/dto/v1_0/converter/ChannelDTOConverter.java:46]: We need to check that the user has the membership on the site related to the Channel.

So, if `commerceChannel.getType()` is site type, we check that the user has the membership on `commerceChannel.getSiteGroupId()`

**Review by alessiorendina — CHANGES_REQUESTED** [review]
Hey @andrea-ale-sbarra , please check my comment. Thanks!

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Pull request closed without merging**